docs/guides/qpu-information.mdx

```diff
@@ -57,24 +57,31 @@ These values are shown on both the [Compute resources page](https://quantum.clou
 Note that the metric for median ECR error (for Eagle processors) and median CZ error (for Heron processors) is the [average gate fidelity](/docs/api/qiskit/quantum_info#average_gate_fidelity), which is the average over all possible input states of the fidelity between the state produced by the actual operation and the state produced by the ideal operation.
 </Admonition>
 
-**Additional information available on the details page for each QPU**
+## Additional information available on the details page for each QPU
 
-To access the details page, click the **Compute resources** tab, then click the name of the QPU.
+To access the details page, select [Compute resources](https://quantum.cloud.ibm.com/computers) from the left-hand menu, then click the name of the QPU.
+
+<span id="native-gates-on-platform"></span>
+* **Native gates** - Each [processor family](/docs/guides/processor-types) has a native gate set. By default, the QPUs in each family only support running the gates and operations in the native gate set. Thus, every gate in the circuit must be translated (by the transpiler) to the elements of this set. To find the native gates for a QPU, select its name on the [Compute resources](https://quantum.cloud.ibm.com/computers) page. The default gates for that QPU are listed under Details. Note that the non-unitary operations are not listed here; use the [method in Qiskit](/docs/guides/get-qpu-information#native-gates-with-qiskit) to see all native gates and operations for a QPU. See a list of all native gates in [this table](#native-gates).
+
+    <Admonition type="note" title="Native gates and basis gates">
+    The terms native gates and basis gates are often used interchangeably. The native gates are those that calibrations have been done for on the QPU, so the QPU can understand and execute them without further instruction. However, you can pass a subset of the native gates as the basis gates to the transpiler. For information about changing the basis gates, see the [Represent quantum computers](/docs/guides/represent-quantum-computers#basis-gates) topic.
+    </Admonition>
```

Comment:
I think we should just delete this, since the basis gates listed on IQP are actually native gates (i.e. you can't change them).